File: DuDoong-Infrastructure/src/main/java/band/gosrock/infrastructure/outer/api/oauth/dto/KakaoTokenResponse.java
package band.gosrock.infrastructure.outer.api.oauth.dto;


import com.fasterxml.jackson.databind.PropertyNamingStrategies.SnakeCaseStrategy;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@JsonNaming(SnakeCaseStrategy.class)
public class KakaoTokenResponse {
    private String accessToken;
    private String refreshToken;
    private String idToken;
}

File: DuDoong-Infrastructure/src/main/kotlin/band/gosrock/infrastructure/outer/api/oauth/dto/KakaoTokenResponse.kt
package band.gosrock.infrastructure.outer.api.oauth.dto

import com.fasterxml.jackson.databind.PropertyNamingStrategies.SnakeCaseStrategy
import com.fasterxml.jackson.databind.annotation.JsonNaming

@JsonNaming(SnakeCaseStrategy::class)
class KakaoTokenResponse {
    var accessToken: String? = null
    var refreshToken: String? = null
    var idToken: String? = null
}
